src/persistence/index.ts

```diff
@@ -1,10 +1,6 @@
-import { HermezTransaction, Exit } from "../domain/hermez";
+/**
+ * HermezJS type definitions have been moved to the dedicated definitions file hermezjs-typings.d.ts
+ * We may decide not to export them from the persistence and let other layers import them from the lib.
+ */
 
-export interface HistoryTransactions {
-  transactions: HermezTransaction[];
-  pendingItems: number;
-}
-export interface HistoryExits {
-  exits: Exit[];
-  pendingItems: number;
-}
+export type { Transactions, Exits, Accounts } from "@hermeznetwork/hermezjs";
```

Comment:
> First of all, I'm starting to think that this persistence module is starting to make no sense by the moment

Absolutely. That is why I added the comment at the top of the file. I was though thinking of keeping all these types in domain and persistence for now, till we finish evolving the typings of the library, to finally removes these modules if they are really not yet required.

For example, in the domain, I'm extending the Deposit interface provided by the types of the lib, since the FE is adding an extra prop there to keep the TX id.

```typescript
export type Deposit = HermezJSDeposit & {
  transactionId?: string;
};
```

Does it make sense to you?

Reply:
Oh I see! Didn't understand the comment at the beginning. Then it's perfect, I would remove this file then.